guides/caching.md

```diff
@@ -217,6 +241,10 @@ The response was not cached because the request body was more than 8000 bytes.
 
 The response was not cached because it contained a `set-cookie` header. To cache the response, use `removeUpstreamResponseHeader('set-cookie')` to remove the set-cookie header.
 
+### deployment
+
+The response was not cached because it was received during the brief time (less than 1 minute) that the new version was being propagated through the global network of POPs. There is no need to take any action because as soon as the new version is completely propagated this status goes away.
```

Comment:
```suggestion
The response was not cached because it was received during the brief time (less than 1 minute) that a new version of the app was being propagated through the global network of POPs. There is no need to take any action because as soon as the new version is completely propagated this status goes away.
```